vxquery-cli/src/main/java/org/apache/vxquery/cli/VXQuery.java

```diff
@@ -240,13 +240,16 @@ private void runQueries() throws IOException, SystemException, Exception {
      * @throws Exception
      */
     private String[] getNodeList() throws Exception {
-        Map<String, NodeControllerInfo> nodeControllerInfos = hcc.getNodeControllerInfos();
-        String[] nodeList = new String[nodeControllerInfos.size()];
-        int index = 0;
-        for (String node : nodeControllerInfos.keySet()) {
-            nodeList[index++] = node;
+        if(hcc != null) {
```

Comment:
Please format the code according to our standard. You can load the eclipse format file into your IDE.